tests/unit/README.rst

```diff
@@ -0,0 +1,48 @@
+******************
+FogLAMP Unit Tests
+******************
+
+Unit tests are the first category of test in FogLAMP. These test ensures that every single function of your code under
+test is generating the desired output.
+
+FogLAMP unit tests heavily use test doubles to replace a production object. A typical example is a code fragment that
```

Comment:
This file itself is in the unit parent directory yet it holds details on python unit tests (mocking in particular). Python specific information should be moved into the python subdirectory